Lib/test/test_asyncio/test_events.py

```diff
@@ -823,6 +823,28 @@ def test_create_server(self):
         # close server
         server.close()
 
+    def test_create_server_trsock(self):
+        proto = MyProto(self.loop)
+        f = self.loop.create_server(lambda: proto, '0.0.0.0', 0)
+        server = self.loop.run_until_complete(f)
+        self.assertEqual(len(server.sockets), 1)
+        sock = server.sockets[0]
+        host, port = sock.getsockname()
+        self.assertEqual(host, '0.0.0.0')
+        dup = sock.dup()
+        self.addCleanup(dup.close)
+        self.assertIsInstance(dup, socket.socket)
+        self.assertFalse(sock.get_inheritable())
```

Comment:
I have some questions about some seemingly random assertions in this test. To begin: what guarantees that the socket is not inheritable? Just that Python sets the default close-on-exec behavior? Should we also check that the dup *is* inheritable? (The man page seems to imply this, at least for [Linux](https://www.man7.org/linux/man-pages/man2/dup.2.html); the macOS man page doesn't mention this.)

Reply:
> what guarantees that the socket is not inheritable? Just that Python sets the default close-on-exec behavior?

Yes, `asyncio` transport sockets should have the same behavior as that of regular sockets. 

> Should we also check that the dup is inheritable?

I don't think that's necessary. I am specifically trying to cover function in https://github.com/python/cpython/blob/main/Lib/asyncio/trsock.py